tests/net/lib/tls_credentials/prj_trusted_tfm_csr.conf

```diff
@@ -0,0 +1,14 @@
+
+# Same as prj_trusted_tfm, but enable CSR generation.
```

Comment:
This must've been overlooked in previous case, but since it's an overlay file it would be good to stick to common overlay file format (`overlay-trusted-tfm-csr.conf`).